frontend/public/components/storage-class-form.tsx

```diff
@@ -55,7 +57,113 @@ export class StorageClassForm_ extends React.Component<
     this.previousName = '';
   }
 
-  storageTypes = Object.freeze({
+  defaultProvisionerObj = {
+    title: '',
+    provisioner: '',
+    parameters: {},
+  };
+
+  CSIStorageTypes = {
+    'rbd.csi.ceph.com': {
+      title: 'Ceph RBD',
+      provisioner: 'rbd.csi.ceph.com',
+      documentationLink: 'https://rook.github.io/docs/rook/master/ceph-block.html',
+      parameters: {
+        clusterID: {
+          name: 'Cluster ID',
+          hintText: 'Cluster ID is the namespace where Ceph is deployed.',
```

Comment:
Just a couple of nitpick notes for all of the hint text properties to maintain consistency with the hint text from the other provisioners:
- Drop the period at the end
- Instead of using the format "`<Parameter name>` is` <description>`", just use the description. 

For example: 
'Cluster ID is the namespace where Ceph is deployed.'  --> 'The namespace where Ceph is deployed'